.github/actions/setup-rust/action.yml

```
@@ -4,19 +4,34 @@ inputs:
+    description: Build OpenBSD std library for cross-compilation
+    required: false
+    default: 'false'
 runs:
   using: composite
   steps:
-    - name: Install rust
-      uses: actions-rust-lang/setup-rust-toolchain@9d7e65c320fdb52dcd45ffaa68deb6c02c8754d9
-      with:
-        override: true
-        components: rustfmt, clippy, llvm-tools-preview
+      - name: Install rust
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 6

(indentation)